public/app/features/alerting/unified/state/actions.ts

```diff
@@ -385,17 +388,49 @@ export function deleteRuleAction(
   };
 }
 
+export const isRulerGrafanaRuleDTO = (rule: RulerRuleDTO): rule is RulerGrafanaRuleDTO => 'grafana_alert' in rule;
+
+// This method was introduced for making betterer happy (it was complaining about using as StoreState)
+const isStoreState = (something: unknown): something is StoreState => {
+  if (typeof something === 'object') {
+    return something ? 'unifiedAlerting' in something : false;
+  } else {
+    return false;
+  }
+};
```

Comment:
Or simpler yet:

```suggestion
const isStoreState = (something: unknown): something is StoreState => typeof something === 'object' && 'unifiedAlerting' in something;
```

Reply:
it needs to be checked if it's null

Reply:
maybe this could work though:
```
const isStoreState = 
(something: unknown): something is StoreState  => typeof something === 'object' && something ? 'unifiedAlerting' in something : false;
```

Reply:
But I think that maybe this 'one line expression' it's more confusing? WDT? @VikaCep @gillesdemey

Reply:
Oh right, `null` is of type object 😮‍💨 Well don't worry then. It was just a nitpick.